jbpm-wb-process-instance-migration/pim-react/src/component/tabMigrationPlan/MigrationPlansEditPopup.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+import { Button } from "patternfly-react";
+import { Modal } from "patternfly-react";
+import { Icon } from "patternfly-react";
+import { OverlayTrigger } from "patternfly-react";
+import { Tooltip } from "patternfly-react";
+export default class MigrationPlansEditPopup extends React.Component {
+  constructor() {
+    super();
+    this.state = { showEditPlanPopup: false };
+  }
+  closeEditPlanPopup = () => {
+    this.setState({ showEditPlanPopup: false });
+  };
+  openEditPlanPopup = () => {
+    this.setState({ showEditPlanPopup: true });
+  };
+
+  submit = () => {
+    //this component is used as "Import Plan"
+    if (this.props.actionName == "Import Plan") {
```

Comment:
Fixed. The comment was bad because this piece of code was used in "Import/Add/Export" plan before, I updated the code and comments to show only "Import/Export" function now